privacyidea/lib/periodictask.py

```diff
@@ -134,6 +175,41 @@ def get_periodic_tasks(name=None, node=None, active=None):
     return result
 
 
+def get_periodic_task_by_name(name):
+    """
+    Get a periodic task by name. Raise ParameterError if the task could not be found.
+    :param name: task name, unicode
+    :return: dictionary
+    """
+    periodic_tasks = get_periodic_tasks(name)
+    if len(periodic_tasks) != 1:
+        raise ParameterError("The periodic task with unique name {!r} does not exist".format(name))
```

Comment:
In theory this message could be misleading, if there where two tasks with the same unique name....?

Reply:
Hm, there hopefully shouldn't be, because we have a ``unique=True`` constraint on the ``name`` field.

Reply:
If the name is unique, then the if statement will be ``<1`` and not ``!=1``.
So the code was misleading to me.
But if the name is always unique, then the message is right.
Leave it this way!